tests/unittests/session_events_tests.py

```
@@ -46,3 +54,201 @@ def test_select_labels(self):
+        state = StateDescription()
+        sample_ids = ["a" * 24, "b" * 24]
+        meta = {
+            "a" * 24: {"type": "default"},
+            "b" * 24: {"type": "alt"},
+        }
+
+        event = SelectSamples(sample_ids=sample_ids, meta=meta)
+        state.selected = event.sample_ids
+        state.selected_meta = event.meta
+
+        self.assertListEqual(state.selected, sample_ids)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertListEqual`

(PT009)